addons/mail/models/mail_thread.py

```diff
@@ -239,7 +239,7 @@ def _search_message_has_error(self, operator, operand):
 
     @api.multi
     def _compute_message_attachment_count(self):
-        read_group_var = self.env['ir.attachment'].read_group([('res_model', '=', self._name)],
+        read_group_var = self.env['ir.attachment'].read_group([('res_field', '=', False), ('res_model', '=', self._name)],
```

Comment:
I think this should be improved also adding `('res_id', 'in', self.ids)`

Reply:
I thought same but this was not needed to solve Issue. (This reminds me Odoo stable Policy)

Reply:
Well, this is very obvious as performance patch, so it can fit inside stable policy, and more being introduced only a few days ago.